packages/katello/katello/katello-remove

```diff
@@ -181,12 +196,20 @@ puts "  * All Katello and Foreman Packages\n"
 puts "Once these packages and configuration files are removed there is no going back."
 puts "If you use this system for anything other than Katello and Foreman you probably"
 puts "do not want to execute this script.\n"
-
-print "Read the source for a list of what is removed.  Are you sure(Y/N)? "
+puts "\n"
+puts "Read the source at /usr/sbin/katello-remove for a list of what is removed.  Are you sure(Y/N) "
```

Comment:
can do
`puts "\nRead the source at /usr/sbin/katello-remove for a list of what is removed.  Are you sure(Y/N) "`
and remove the single puts "\n"